### PR TITLE
Add Grafana dashboard discovery info

### PR DESCRIPTION
**Pull request by fredtibbitts**
This PR adds instructions to the monitoring metrics page for Grafana dashboard discovery.

### CODE REVIEW

**Review by davisgomes — APPROVED** [review]

File: content/en/docs/monitoring/metrics/metrics.md [hunk 1]
```diff
@@ -203,3 +203,36 @@ To view the application metrics for the `todo-list` example application, select
 because the `todo-list` application is a WebLogic application.
 
 ![WebLogicDashboard](/docs/images/grafana-weblogic-dashboard.png)
+
+### Dashboard discovery
+
+The Verrazzano Grafana instance supports dynamic dashboard discovery. This allows users to deploy dashboards along with other application components.
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/metrics/metrics.md:209]: -> This lets you deploy dashboards

File: content/en/docs/monitoring/metrics/metrics.md [hunk 2]
```diff
@@ -203,3 +203,36 @@ To view the application metrics for the `todo-list` example application, select
...
+
+Grafana will automatically discover dashboards in ConfigMaps that are labeled with `grafana_dashboard: "1"`. The ConfigMap must contain the dashboard JSON.
+The ConfigMap may also be annotated with `k8s-sidecar-target-directory` to specify the name of a Grafana folder.
+
+Here is an example ConfigMap.
+
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/metrics/metrics.md:215]: If you want users to be able to copy this ConfigMap, please put it between the Hugo copy button code:
{{< clipboard >}}
<div class="highlight">
...
</div>
{{< /clipboard >}}

**Review by rosemarymarano — COMMENTED** [review]
Minor edits

File: content/en/docs/monitoring/metrics/metrics.md (hunk 1 above)
Comment (fredtibbitts) [reply content/en/docs/monitoring/metrics/metrics.md]: Fixed.

File: content/en/docs/monitoring/metrics/metrics.md (hunk 2 above)
Comment (fredtibbitts) [reply content/en/docs/monitoring/metrics/metrics.md:215]: This example only contains a snippet of the full ConfigMap (which would be very large), so I don't think it makes sense to have a copy button here.

Comment (rosemarymarano) [reply content/en/docs/monitoring/metrics/metrics.md:215]: ok, makes sense.

**Review by rosemarymarano — APPROVED** [review]

**Review by desagar — APPROVED** [review]

**Pull request merged**